package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@jupiterone/jupiterone-client-nodejs",
-  "version": "0.25.1",
+  "version": "0.25.2",
```

Comment:
"Added" section in CHANGELOG, but patch version. 🤔 🤔 🤔

Reply:
I was unaware of the great `0.x.y` argument until recently. Where I'm at now is, on average, developers perceive `minor` upgrades to mean `breaking changes` before version `1.0.0`. I think this should've been moved to `0.26.0` because it actually included breaking changes to the `list` method. The jury is still out on how we want to increment versions pre `1.0.0`, but that's where my head is at FWIW.

A glimpse into the debate: https://github.com/semver/semver/issues/221#issue-41992376

Reply:
Ah, that's a good point. I didn't think about the fact that this is a "pre-release" package (still).